src/__tests__/apis/UserApi.test.ts

```diff
@@ -58,6 +58,20 @@ test('getUserList() with limit returns a list of users', async () => {
   expect(userList[0]).toBeInstanceOf(User);
 });
 
+test('getUserList() with emails and phone numbers returns a list of users', async () => {
+  nock('https://api.clerk.dev')
+      .get('/v1/users?email_address=email1&email_address=email2&phone_number=phone1&phone_number=phone2')
+      .replyWithFile(200, __dirname + '/responses/getUserList.json', {
+        'Content-Type': 'application/x-www-form-urlencoded',
+      });
+
+  const userList = await users.getUserList({ emailAddress: ['email1', 'email2'], phoneNumber: ['phone1', 'phone2'] });
```

Comment:
I'm guessing that filtering by a single email or phone would make more sense here.

It's not evident that the server would use AND or OR for these conditions.

Reply:
I see what you mean, but I think filtering for multiple emails is a valid scenario, no?

Reply:
Usually a query string of `email=foo@bar.com&email=yolo@swag.com` would imply an AND condition which in this case would not yield any results.

Reply:
Actually, I would say that usually, it's the other way around. 
Query parameters which allow multiple values are represented like this, i.e. `id=value1&id=value2`, and usually this implies an OR condition between the values. 

In any case, I've added some clarification in README.